packages/react-form-renderer/rollup.config.js

```diff
@@ -29,27 +32,41 @@ function onwarn(warning) {
   throw Error(warning.message);
 }
 
-export default [{
+const plugins = [
+  nodeResolve(),
+  babel(babelOptions),
+  commonjs(commonjsOptions),
+  nodeGlobals(), // Wait for https://github.com/cssinjs/jss/pull/893
```

Comment:
what is this comment? :eyes: The mentioned PR has been merged!

Reply:
Oh right this was due to SSR. We should keep that one. I will just remove the comment